Source/WebCore/page/Quirks.cpp

```diff
@@ -496,11 +496,9 @@ bool Quirks::shouldDispatchSimulatedMouseEvents(const EventTarget* target) const
         return false;
 
     case ShouldDispatchSimulatedMouseEvents::DependingOnTargetFor_mybinder_org:
-        if (is<Node>(target)) {
-            for (const auto* node = downcast<Node>(target); node; node = node->parentNode()) {
-                if (is<Element>(node) && const_cast<Element&>(downcast<Element>(*node)).classList().contains("lm-DockPanel-tabBar"_s))
-                    return true;
-            }
+        for (RefPtr node = dynamicDowncast<Node>(target); node; node = node->parentNode()) {
+            if (RefPtr element = dynamicDowncast<Element>(node); element && const_cast<Element&>(*element).classList().contains("lm-DockPanel-tabBar"_s))
```

Comment:
@cdumez I suspect that if I make this `auto*` again it'll work for GCC. Thoughts?